### PR TITLE
Gpl 101 pipelines documentation

### PR DESCRIPTION
**Pull request by stevieing**

### CODE REVIEW

File: app/pipelines/pipelines/requestor/factory.rb [hunk 1]
```diff
@@ -14,34 +17,51 @@ module Factory
...
+      # checks if the factory is valid, if so will save all of the requests
+      # @return [Boolean] if the requests have been successfully saved or not
       def save
         return false unless valid?
 
         requests.collect(&:save)
         true
       end
 
+      # Takes each set of request attributes:
```
Comment (harrietc52) [inline app/pipelines/pipelines/requestor/factory.rb]: These bullet points aren't quite working I don't think

File: app/pipelines/pipelines/requestor/factory.rb [hunk 2]
```diff
@@ -51,6 +71,9 @@ def build_requests(attributes)
         end
       end
 
+      # Validates the requests:
```
Comment (harrietc52) [inline app/pipelines/pipelines/requestor/factory.rb]: And these

File: app/pipelines/pipelines/requestor/model.rb [hunk 3]
```diff
@@ -3,6 +3,12 @@
 module Pipelines
   module Requestor
     # Model - behaviour for pipeline requests model
+    # includes various bits of behaviour in the model:
+    # * the Material concern
+    # * request association
+    # * sample association
+    # * delegation of sample attributes readers
+    # * validation to check that each of the request attributes is present
```
Comment (harrietc52) [inline app/pipelines/pipelines/requestor/model.rb]: These are though!

File: app/pipelines/pipelines/requestor/factory.rb (hunk 2 above)
Comment (stevieing) [reply app/pipelines/pipelines/requestor/factory.rb]: so are these ...

Comment (harrietc52) [reply app/pipelines/pipelines/requestor/factory.rb]: Ok can leave it then, happy to merge

**Pull request merged**